app/javascript/mastodon/components/collapsable.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { Motion, spring } from 'react-motion';
+import Motion from 'react-motion/lib/Motion';
+import spring from 'react-motion/lib/spring';
```

Comment:
This doesn't appear to be part of [the public API per the docs](https://github.com/chenglou/react-motion) but if it reduces the bundle size, I'm in. :) It may need to be reverted if they release a breaking change, though.